ingestion/src/metadata/ingestion/ometa/routes.py

```diff
@@ -332,4 +336,6 @@
     DriveService.__name__: "/services/driveServices",
     CreateDriveServiceRequest.__name__: "/services/driveServices",
     EntityProfile.__name__: "/entity/profiles",
+    DynamicAgent.__name__: "/agents/dynamic",
+    CreateChatConversation.__name__: "/assistants/chatConversations",
```

Comment:
@TeddyCr I think DynamicAgent is a collate entity. We might need to pass this to the collate extended routes?